### PR TITLE
New Recipe: Antic v0.0.1+0

### PR DESCRIPTION
**Pull request by thofma**

### CODE REVIEW

File: A/Antic/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Antic"
+version = v"0.0.1+0"
```
Comment (giordano) [inline A/Antic/build_tarballs.jl:6]: You can't specify a build version:
```suggestion
version = v"0.0.1"
```
Is really this the upstream version number?

**Comment by thofma** [comment]
Thanks!

File: A/Antic/build_tarballs.jl (hunk 1 above)
Comment (aviks) [reply A/Antic/build_tarballs.jl]: What Mose was saying is the the version here should generally be the same as the underlying library. It looks like the latest release of antic is 0.2, but you have built the master as it stands today. I'd suggest calling it 0.2.0, and then updating it as new releases are tagged in antic.

Comment (thofma) [reply A/Antic/build_tarballs.jl]: Thanks, I have updated the version number.

**Comment by thofma** [comment]
Merge?

**Comment by giordano** [comment]
https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=4065&view=results
```
┌ Error: Unable to find valid license file in "${prefix}/share/licenses/Antic"
└ @ BinaryBuilder /depot/packages/BinaryBuilder/W9GL3/src/Auditor.jl:438
```
See https://juliapackaging.github.io/BinaryBuilder.jl/dev/build_tips/#Installing-the-license-file-1

**Comment by thofma** [comment]
All the green was misleading. I have added the license (upstream) and updated the commit.

**Comment by fingolfin** [comment]
@thofma now that Antic 0.2.1 is tagged and released, I assume you'll update this once more? Thank you :-)

**Comment by giordano** [comment]
You mean #1156?

**Comment by fingolfin** [comment]
Awesome, @thofma rocks :-)

**Pull request merged**